proposals/16:upgrade-8/sanity.test.js

```diff
@@ -0,0 +1,30 @@
+/** @file adapted from upgrade-9's sanity, ensure addresses are what we expected */
+import test from 'ava';
+
+import {
+  GOV1ADDR,
+  GOV2ADDR,
+  GOV3ADDR,
+  USER1ADDR,
+  VALIDATORADDR,
+} from '@agoric/synthetic-chain/src/lib/constants.js';
```

Comment:
now that they're actually constants, `constants.js` shouldn't get them from the environment.

Testing that the environment variables are set correctly is still worthwhile. This test file could do that.